src/com/palmergames/bukkit/towny/utils/PlayerCacheUtil.java

```diff
@@ -242,6 +242,10 @@ public static void cacheBlockErrMsg(Player player, String msg) {
 	 * @return {@link TownBlockStatus} which will be used by Towny.
 	 */
 	public static TownBlockStatus fetchTownBlockStatus(Player player, WorldCoord worldCoord) {
+		
+		if (TownySettings.isFakeResident(player.getName()))

```

Comment:
Might be nice to have this support uuids, in cases where mods don't use []'s but do have a hardcoded uuid.

Reply:
Yeah this one needs to be fleshed out.